Project.toml

```diff
@@ -0,0 +1,30 @@
+name = "ImageSegmentation"
+uuid = "80713f31-8817-5129-9cf8-209ff8fb23e1"
+version = "1.2.0"
+
+[deps]
+Clustering = "aaaa29a8-35af-508c-8bc3-b662a17a0fe5"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
+LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+RegionTrees = "dee08c22-ab7f-5625-9660-a9af2021b33f"
+SimpleWeightedGraphs = "47aef6b3-ad0c-573a-a1e2-d07658019622"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+[compat]
+Clustering = ">= 0.9.0"
+DataStructures = ">= 0.7.1"
+ImageFiltering = ">= 0.1.3"
+Images = ">= 0.9"
+LightGraphs = ">= 0.9.0"
+RegionTrees = ">= 0.1.0"
+SimpleWeightedGraphs = ">= 0.0.1"
+StaticArrays = ">= 0.5.0"
+julia = ">= 0.7"
+
+[extras]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Needs a `targets` section

Reply:
sometimes targets section get accidentally deleted when add/rm packages

Reply:
Hmm, I think I may have noticed it too, though I didn't remember that.